### DISCUSSION TITLE
uhhh 1.4.2 thing HELP

### DISCUSSION THREAD

**Discussion by SuperSpinDashGamer39** [KadeArchive/Kade-Engine#2396] [Q&A]
I'm trying to make custom notes but here is the thing IM ON KADE ENGINE 1.4.2 and most of the tutorials I click are earther done on a different engine or done on 1.5.4 pls help

**Comment by SuxxorState** [comment]
is there a reason why you're on 1.4? im not judging im just curious
as for the notes, i have no clue how to help cause ive never experimented with anything before 1.5
